controls/combobox/databinding.md

```diff
@@ -11,7 +11,9 @@ slug: combobox-databinding
 - `ItemsSource`(`IEnumerable`)&mdash;Defines the collection of the items that will populate the control with data.
 - `DisplayMemberPath`(`string`)&mdash;Defines the name of the property which will be visualized inside the drop-down list.
 
-> If DisplayMemberPath is not set the “ToString” implementation of the business object will be visualized. The DisplayMemberPath is a property that helps the developers to visualize an exact property from the business object they are bound to.
+> If `DisplayMemberPath` is not set the `ToString` implementation of the business object will be visualized. The `DisplayMemberPath` is a property that helps the developers to visualize an exact property from the business object they are bound to. 
```

Comment:
* Avoid placing two notes next to each other because their effectiveness diminishes. The same happens with notes next to lists with no text paragraphs in between.
* Let's first explain what the `DisplayMemberPath` is, and then mention what happens when we don't use it.
```suggestion
The `DisplayMemberPath` is a property that lets the developer specify a particular property of the business object to be displayed. If `DisplayMemberPath` is not set, the ComboBox will visualize the `ToString` implementation of the business object. 
```